docs/source/guides/installation_guide.md

```diff
@@ -142,36 +158,70 @@ Install the Solidity compiler (solc):
 ```bash
 $(nucypher) pipenv run install-solc
 ```
-    
+
 ### Pip Development Installation
-    
+
 Alternately, you can install the development dependencies with pip:
 
 ```bash
 $ pip3 install -e .[development]
 $ ./scripts/installation/install_solc.sh
 ```
 
+### Development Docker Installation
+The intention of the Docker configurations in this directory is to enable anyone to develop and test NuCypher on all major operating systems with minimal prerequisites and installation hassle.
+
+#### Start with standard Docker Installation
+1. Install [Docker](https://docs.docker.com/install/)
+2. Install [Docker Compose](https://docs.docker.com/compose/install/)
+3. `cd` to `dev/docker
+4. Run `docker-compose up --build` **this must be done once to complete install**
+
+#### Running NuCypher
+Then you can do things like:
+* Run the tests:
+`docker-compose run nucypher-dev pytest`
+* Start up an Ursula:
+`docker-compose run nucypher-dev nucypher ursula run --dev --federated-only`
+* Open a shell:
+`docker-compose run nucypher-dev bash`
+
+* try some of the scripts in `dev/docker/scripts/`
+
+**tested on (Ubuntu 16, MacOS 10.14, Windows 10)*
```

Comment:
Are the asterisks supposed to be mismatched here?

Reply:
✅